sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert str((year/week).n(2)) == '52.'
```

Comment:
Good question. No, a year is not exactly 52 weeks. Some relevant output:

>>> year / week
36524219/700000
>>> (year / week).evalf()
52.1774557142857